### DISCUSSION TITLE
ArrowDropDownIcon

### DISCUSSION THREAD

**Discussion by Min121e** [tailwindlabs/heroicons#1008] [Ideas]
[![image](https://github.com/tailwindlabs/heroicons/assets/97957887/00eab313-b943-40b7-a0d1-b532bfa6faea)
](https://cdn-icons-png.flaticon.com/512/25/25243.png)

please make icon like this.

**Comment by jlous** [comment]
Ideally a matching set of flat, solid triangles to complement the chevrons.
